Source/WTF/wtf/CompactVariantOperations.h

```diff
@@ -90,6 +97,19 @@ template<CompactVariantAlternative... Ts> struct CompactVariantOperations {
         else
             new (NotNull, &data) T(std::forward<U>(payload));
 
+        // For data other than pointers, ensure the top byte is zero.
+        // (pointers may store arbitrary data in the top byte, and that's okay due to TBI)
+        if constexpr (!CompactVariantAlternativePointer<T>) {
+            // Sanity check to make sure the data doesn't overwrite the top byte.
+            RELEASE_ASSERT(!(data & topByteMask));
+            data &= ~topByteMask;
```

Comment:
This isn't needed. The `RELEASE_ASSERT` guarantees it's a no op.

Reply:
I've removed it.